utils/bitcoin.command.helpers.ts

```diff
@@ -95,25 +95,61 @@ export const displayAndConfirmTransaction = async (info: TransactionInfo) => {
 
   console.log(`
 Network: ${info.network}
-${info.amount ? `Amount: ${info.amount} BTC` : ""}
+${
+  info.amount
+    ? `Amount: ${info.amount} BTC (${ethers.parseUnits(info.amount, 8)} sats)`
+    : ""
+}
+Inscription Commit Fee: ${info.inscriptionCommitFee} sats
+Inscription Reveal Fee: ${info.inscriptionRevealFee} sats
+Deposit Fee: ${info.depositFee} sats
+Total: ${
+    Number(ethers.parseUnits(info.amount, 8)) +
+    info.inscriptionCommitFee +
+    info.inscriptionRevealFee +
+    info.depositFee
+  } sats
 Gateway: ${info.gateway}
 Sender: ${info.sender}
-Universal Contract: ${info.receiver || notApplicable}
+Receiver: ${info.receiver || notApplicable}
 Revert Address: ${info.revertAddress || notApplicable}
 Operation: ${info.operation}
 ${info.encodedMessage ? `Encoded Message: ${info.encodedMessage}` : ""}
 Encoding Format: ${info.encodingFormat}
 Raw Inscription Data: ${info.rawInscriptionData}
-Fees:
-  - Commit Fee: ${info.commitFee} sat
-  - Reveal Fee: ${info.revealFee} sat
-  - Total Fee: ${info.totalFee} sat (${(info.totalFee / 100000000).toFixed(
-    8
-  )} BTC)
 `);
   await confirm({ message: "Proceed?" }, { clearPromptOnDone: true });
 };
 
+/**
+ * Displays memo transaction details to the user and asks for confirmation before proceeding
+ */
+export const displayAndConfirmMemoTransaction = async (
+  amount: number,
+  networkFee: number,
+  depositFee: number,
+  gateway: string,
+  sender: string,
+  memo: string
+) => {
+  console.log(`
+Network: Signet
+Gateway: ${gateway}
+Sender: ${sender}
+Operation: Memo Transaction
+Memo: ${memo}
+Deposit Amount: ${amount} sats (${(amount / 100000000).toFixed(8)} BTC)
+Network Fee: ${networkFee} sats (${(networkFee / 100000000).toFixed(8)} BTC)
+Deposit Fee: ${depositFee} sats (${(depositFee / 100000000).toFixed(8)} BTC)
+Deposit Total: ${amount + depositFee} sats (${(
+    amount / 100000000 +
+    depositFee / 100000000
+  ).toFixed(8)} BTC)
```

Comment:
It seems like we're repeating this number a lot, we should extract it as a constant.

Reply:
Added a small helper function instead: 3fae7ad